test/e2e/suite/scenario/quickstart.go

```diff
@@ -0,0 +1,84 @@
+	"github.com/notaryproject/notation/test/e2e/internal/utils/validator"
+	. "github.com/onsi/ginkgo/v2"
+)
+
+// quickstart doc: https://notaryproject.dev/docs/quickstart/
+var _ = Describe("notation quickstart E2E test", Ordered, func() {
+	var vhost *utils.VirtualHost
+	var artifact *Artifact
+	var artifact2 *Artifact
+	var notation *utils.ExecOpts
+	BeforeAll(func() {
+		var err error
+		// setup host
+		vhost, err = utils.NewVirtualHost(NotationBinPath, CreateNotationDirOption())
+		if err != nil {
+			panic(err)
+		}
+		vhost.SetOption(AuthOption("", ""))
+		notation = vhost.Executor
+
+		// add an image to the OCI-compatible registry
+		artifact = GenerateArtifact("", "")
+		artifact2 = GenerateArtifact("", "")
+	})
+
+	It("list the signatures associated with the container image", func() {
+		notation.Exec("ls", artifact.ReferenceWithTag()).
+			MatchContent("")
+	})
+
+	It("generate a test key and self-signed certificate", func() {
+		notation.Exec("cert", "generate-test", "--default", "wabbit-networks.io").
+			MatchKeyWords(
+				"Successfully added wabbit-networks.io.crt",
+				"wabbit-networks.io: added to the key list",
+				"wabbit-networks.io: mark as default signing key")
+
+		notation.Exec("key", "ls").
+			MatchKeyWords(
+				"notation/localkeys/wabbit-networks.io.key",
+				"notation/localkeys/wabbit-networks.io.crt",
+			)
+
+		notation.Exec("cert", "ls").
+			MatchKeyWords("notation/truststore/x509/ca/wabbit-networks.io/wabbit-networks.io.crt")
+	})
+
+	It("sign the container image", func() {
+		notation.Exec("sign", artifact.ReferenceWithDigest()).
+			MatchContent(fmt.Sprintf("Successfully signed %s\n", artifact.ReferenceWithDigest()))
+
+		notation.Exec("ls", artifact.ReferenceWithDigest()).
+			MatchKeyWords(fmt.Sprintf("%s\n└── application/vnd.cncf.notary.signature\n    └── sha256:", artifact.ReferenceWithDigest()))
+	})
+	It("sign the container image with cose format", func() {
+		notation.Exec("sign", "--signature-format", "cose", artifact2.ReferenceWithDigest()).
+			MatchContent(fmt.Sprintf("Successfully signed %s\n", artifact2.ReferenceWithDigest()))
+
+		notation.Exec("ls", artifact2.ReferenceWithDigest()).
+			MatchKeyWords(fmt.Sprintf("%s\n└── application/vnd.cncf.notary.signature\n    └── sha256:", artifact2.ReferenceWithDigest()))
+	})
+
+	It("Create a trust policy", func() {
+		vhost.SetOption(AddTrustPolicyOption("quickstart_trustpolicy.json"))
+		validator.CheckFileExist(vhost.AbsolutePath(NotationDirName, TrustPolicyName))
+	})
+
+	It("Verify the container image", func() {
+		notation.Exec("verify", artifact.ReferenceWithDigest()).
+			MatchContent(fmt.Sprintf("Successfully verified signature for %s\n", artifact.ReferenceWithDigest()))
+	})
+
+	It("Verify the container image with cose format", func() {
+		notation.Exec("verify", artifact2.ReferenceWithDigest()).
+			MatchContent(fmt.Sprintf("Successfully verified signature for %s\n", artifact2.ReferenceWithDigest()))
+	})
```

Comment:
This test cannot tell whether JWS fails or COSE fails. It can only tell both fails.

Reply:
As you can see,
```go
It("Verify the container image with cose format", func() {
```
This line explains what is doing and will be shown in the test output.
Also updated the description for verifying JWS signature, so the output can tell whether JWS or COSE is failed.

Reply:
example output
```
• [0.042 seconds]
notation quickstart E2E test
/home/jj/projects/junjie/notation/test/e2e/suite/scenario/quickstart.go:13
  Verify the container image with jws format
  /home/jj/projects/junjie/notation/test/e2e/suite/scenario/quickstart.go:75

  Begin Captured StdOut/StdErr Output >>

    >> should pass: /home/jj/projects/junjie/notation/bin/notation verify localhost:5000/e2e-8223469589761937864@sha256:cc2ae4e91a31a77086edbdbf4711de48e5fa3ebdacad3403e61777a9e1a53b6f >>
    Successfully verified signature for localhost:5000/e2e-8223469589761937864@sha256:cc2ae4e91a31a77086edbdbf4711de48e5fa3ebdacad3403e61777a9e1a53b6f
  << End Captured StdOut/StdErr Output

  Begin Captured GinkgoWriter Output >>
    STEP: 
    >> should pass: /home/jj/projects/junjie/notation/bin/notation verify localhost:5000/e2e-8223469589761937864@sha256:cc2ae4e91a31a77086edbdbf4711de48e5fa3ebdacad3403e61777a9e1a53b6f >> 01/16/23 16:50:07.164
  << End Captured GinkgoWriter Output
------------------------------
notation quickstart E2E test
  Verify the container image with cose format
  /home/jj/projects/junjie/notation/test/e2e/suite/scenario/quickstart.go:80
------------------------------
• [0.032 seconds]
notation quickstart E2E test
/home/jj/projects/junjie/notation/test/e2e/suite/scenario/quickstart.go:13
  Verify the container image with cose format
  /home/jj/projects/junjie/notation/test/e2e/suite/scenario/quickstart.go:80

  Begin Captured StdOut/StdErr Output >>

    >> should pass: /home/jj/projects/junjie/notation/bin/notation verify localhost:5000/e2e-15492302041286846102@sha256:cc2ae4e91a31a77086edbdbf4711de48e5fa3ebdacad3403e61777a9e1a53b6f >>
    Successfully verified signature for localhost:5000/e2e-15492302041286846102@sha256:cc2ae4e91a31a77086edbdbf4711de48e5fa3ebdacad3403e61777a9e1a53b6f
  << End Captured StdOut/StdErr Output

  Begin Captured GinkgoWriter Output >>
    STEP: 
    >> should pass: /home/jj/projects/junjie/notation/bin/notation verify localhost:5000/e2e-15492302041286846102@sha256:cc2ae4e91a31a77086edbdbf4711de48e5fa3ebdacad3403e61777a9e1a53b6f >> 01/16/23 16:50:07.207
  << End Captured GinkgoWriter Output

```